firestartr-bootstrap/firestartr_files/claims/.config/claims_defaults.yaml

```diff
@@ -2,8 +2,10 @@ ComponentClaim:
   version: "1.0"
   type: service
   lifecycle: "production"
-  platformOwner: null
+  platformOwner: group:prefapp-all
```

Comment:
This should be a template and interpolate the real name of the org we are bootstrapping .